### ISSUE TITLE
Upload and processing datasets

### ISSUE THREAD

**Issue by PieterjanVerhelst** [PieterjanVerhelst/eel-pdst-analysis#1]
Pop-off data storage tags log data (temperature and pressure) during an eel's trajectory. After a preprogrammed time, the tag detaches, surfaces, drifts ashore and can be found by, for instance, beachcombers who send the tag back to us. Consequently, each datafile contains information related to the trajectory of 1 eel, logging multiple variables (i.e. temperature and pressure) at different rates (10 s and 2 s, respectively). Also, each file starts with a number of records of metadata, which are redundant for the data exploration and analysis part.
In conclusion, we want to upload all the files (i.e. the trajectories) at once and format them at once as well. I think the end 'product' can be a list of processed dataframes with the following format:

| Date | Time | Temperature | Pressure |
| -- | -- | -- | -- |

Yet again, notice that temperature is measured at 10 s intervals and pressure at 2 s intervals.
